README.md

```diff
@@ -398,7 +398,7 @@ h("a", { class: { active: true, selected: false } }, "Toggle");
 In JSX, you can use `class` like this:
 
 ```jsx
-<div class={{ "foo": true, "bar": true }} />
+<div class={{ foo: true, bar: true }} />
```

Comment:
These kind of changes has been introduced by Prettier kicked by `lint-staged` when I try to make a commit.
Is it okay to commit them? 🤔

Reply:
Yeah, that is fine